data/json/furniture_and_terrain/furniture-appliances.json

```
@@ -567,21 +567,11 @@
     "name": "mounted solar panel",
     "description": "A set of photovoltaic power generators, which turns solar radiation into useable electricity.  While useful before the Cataclysm, they have become priceless tools, invaluable to any survivor.",
     "symbol": "#",
-    "color": "yellow",
+    "color": "light_blue",
```

```suggestion
    "color": "yellow",
```
It's not necessary to change the color